build/sbt

```diff
@@ -68,7 +68,7 @@ realpath () {
 
 # Make Jenkins use Google Mirror first as Maven Central may ban us
 SBT_REPOSITORIES_CONFIG="$(dirname "$(realpath "$0")")/sbt-config/repositories"
-export SBT_OPTS="-Dsbt.override.build.repos=true -Dsbt.repository.config=$SBT_REPOSITORIES_CONFIG"
+export SBT_OPTS="-Dsbt.repository.config=$SBT_REPOSITORIES_CONFIG"
```

Comment:
We need this change to allow the master build to add the apache snapshot repo. Another way of doing this would be to add the snapshot repo to the repositories file.

Reply:
hi @hvanhovell , I think we want to keep the `-Dsbt.override.build.repos=true`, because the `repositories` files has a order of maven repos to try and we prioritize Google maven mirror over the Central maven because of latter's rate limits.

Can we add the Apache snapshots repo to `repositories` file?

Reply:
Done.